install.sh

```diff
@@ -1,25 +1,98 @@
-#!/bin/sh
-
-echo "Installing latest 'mob' release from GitHub"
-case "$(uname -s)" in
-   Darwin)
-      system="darwin"
-     ;;
-   *)
-      system="linux"
-     ;;
-esac
-url=$(curl -s https://api.github.com/repos/remotemobprogramming/mob/releases/latest \
-| grep "browser_download_url.*mob_.*${system}_amd64\.tar\.gz" \
-| cut -d ":" -f 2,3 \
-| tr -d \")
-# echo "$url"
-tarball="${url##*/}"
-
-curl -sSL $url | tar xz -C /usr/local/bin/ mob && chmod +x /usr/local/bin/mob
-
-location="$(which mob)"
-echo "Mob binary location: $location"
-
-version="$(mob version)"
-echo "Mob binary version: $version"
\ No newline at end of file
+#!/bin/sh   
+target=/usr/local/bin
+local_target=$(systemd-path user-binaries)
+user_arg=$1
+stream_cmd="curl -s https://raw.githubusercontent.com/remotemobprogramming/mob/master/install.sh"
+readme_location="https://github.com/remotemobprogramming/mob/blob/master/README.md"
+
+handle_user_installation() {
+  if [ "$user_arg" = "--user" ]
+  then
+    if [ "$local_target" != "" ] && [ ! -d "$local_target" ]
+    then
+      mkdir $local_target
```

Comment:
'mkdir -p'?

Reply:
Should the script fail if mkdir fails? Probably so.

Reply:
if you look down, you'll see that the next command will fail as the directory won't exist.